device-services/device-random/Attribution.txt

```diff
@@ -0,0 +1,148 @@
+The following open source projects are referenced by Device Service Random Go:
```

Comment:
This file can be removed for examples since we don't release the artifact .

Reply:
Removed.